.gitignore

```diff
@@ -131,3 +134,5 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+
+app/routers/stam
```

Comment:
Not sure this should be here. If everyone would list any random file they created just to check something this file would get messy real fast. Just don't commit the file.

Reply:
תכלס